Examples/Image/Detection/FastRCNN/A2_RunWithPyModel.py

```diff
@@ -54,6 +54,13 @@
     pool_node_name = "pool3"
     last_hidden_node_name = "h2_d"
     roi_dim = 6
+elif base_model == "VGG16":
+    model_file = "../../../../../PretrainedModels/VGG16_ImageNet_Caffe.model"
```

Comment:
What is the mAP that you achieve with VGG? Please share a trained model for us to verify, thanks.